### PR TITLE
Copy limited pieces of code we use from runc's apparmor and utils packages

### PR DESCRIPTION
**Pull request by dims**
/kind cleanup

Follow up from discussion in:
https://github.com/kubernetes/kubernetes/issues/128157#issuecomment-2427394891

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by dims** [comment]
cc @haircommander @kolyshkin 

/assign @mrunalp @derekwaynecarr

File: third_party/forked/libcontainer/utils/utils.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package utils
```
Comment (haircommander) [inline third_party/forked/libcontainer/utils/utils.go:1]: is this intentionally unused now?

Comment (dims) [reply third_party/forked/libcontainer/utils/utils.go:1]: E_TOO_MANY_BRANCHES... pushed an update now :)

**Comment by haircommander** [comment]
LGTM, @kubernetes/sig-node-leads  WDYT?

**Review by mrunalp — APPROVED** [review]
/lgtm

**Pull request merged**